crates/walrus-service/src/node/server.rs

```diff
@@ -733,11 +737,13 @@ mod tests {
         (config, handle)
     }
 
-    fn default_client_builder() -> ClientBuilder {
-        Client::builder().no_proxy().tls_built_in_root_certs(false)
+    fn default_client_builder() -> StorageNodeClientBuilder {
```

Comment:
`default_storage_node_client_builder` for consistency?

Reply:
Good call.